File: 백준/Bronze/2920. 음계/음계.js
console.log(solution(require("fs").readFileSync("/dev/stdin").toString().trim()));

function solution(input) {
  if (input === "1 2 3 4 5 6 7 8") return "ascending";
  else if (input === "8 7 6 5 4 3 2 1") return "descending";
  else return "mixed";
}

File: 백준/Bronze/2920. 음계/음계.cc
#include <iostream>
using namespace std;

int main() {
    int l[8];
    string s;
    cin >> l[0] >> l[1] >> l[2] >> l[3] >> l[4] >> l[5] >> l[6] >> l[7];

    for(int i=0; i<7; i++) {
        if(l[i+1] == l[i]+1) {
            s="ascending";
        } else if (l[i+1] == l[i]-1) {
            s="descending";
        } else {
            s="mixed";
            break;
        }
    }

    cout << s << "\n";

    return 0;
}